libs/ui-watt/src/lib/foundations/color/storybook/color.stories.ts

```diff
@@ -0,0 +1,41 @@
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+
+import { ColorComponent } from './color.component';
+import { ColorModule } from './color.module';
+
+export default {
+  title: 'Foundations/Color',
+  component: ColorComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [ColorModule],
+    })
+  ],
+} as Meta<ColorComponent>;
+
+//👇 We create a “template” of how args map to rendering
+const Template: Story<ColorComponent> = (args) => ({
+  props: args,
+});
+
+//👇 Each story then reuses that template
+export const Color = Template.bind({});
+Color.parameters = {
+  docs: {
+    source: {
+      code: ` // Usage from SCSS:
+@use '@energinet/watt' as watt;
+.my-element {
+  background: watt.getColor('COLOR NAME');
+  color: watt.getColorContrast('COLOR NAME-contrast');
```

Comment:
Question: Shouldn't the "COLOR NAME-contrast" part be just "COLOR NAME"?

Nitpick: Consider changing "COLOR NAME" string to "<color-name>" to make it more explicit.